{% include "muttemp.html" %}
{% import "bootstrap/wtf.html" as wtf %}

{% block content %}
<style>
body
{
	background-color:rgba(204,204,204,0.1);
}
img
{
	{#width:80%;#}
    height:auto;
}
</style>
<style>
/* Custom Styles */
    ul.nav-tabs{
        width: 140px;
        margin-top: 20px;
        border-radius: 4px;
        border: 1px solid #ddd;
        box-shadow: 0 1px 4px rgba(0, 0, 0, 0.067);
    }
    ul.nav-tabs li{
        margin: 0;
        border-top: 1px solid #ddd;
    }
    ul.nav-tabs li:first-child{
        border-top: none;
    }
    ul.nav-tabs li a{
        margin: 0;
        padding: 8px 16px;
        border-radius: 0;
    }
    ul.nav-tabs li.active a, ul.nav-tabs li.active a:hover{
        color: #fff;
        background: #0088cc;
        border: 1px solid #0088cc;
    }
    ul.nav-tabs li:first-child a{
        border-radius: 4px 4px 0 0;
    }
    ul.nav-tabs li:last-child a{
        border-radius: 0 0 4px 4px;
    }
    ul.nav-tabs.affix{
        top: 30px; /* Set the top position of pinned element */
    }
</style>
<script>
$(document).ready(function(){
    $("#myNav").affix({
        offset: {
            top: 125
      }
    });
});
</script>
<div class="container" data-spy="scroll" data-target="#myScrollspy">

        <div>
            <div class="col-md-3" id="myScrollspy">
            <ul class="nav nav-tabs nav-stacked" id="myNav">
                <li class="active"><a href="#collapse1">1.Overview</a></li>
                <li><a href="#collapse2">2.Data Summary</a></li>
{#                <li><a href="#collapse3">3.Browse</a></li>#}
                <li><a href="#collapseFive">Protein-Protein Interaction</a></li>
            </ul>
        </div>

            <div class="col-md-9">

                <h2 id="collapseOne">1.Overview</h2>
                <p><b>PsyMuKB</b> (<b><u>Psy</u></b>chiatric <b><u>Mu</u></b>tation <b><u>K</u></b>nowledge <b><u>B</u></b>ase) is a comprehensive knowledge base that provides information about various types of <u><b><i>de novo</i></b> variants</u>, such as single nucleotide variations (<b><u>SNVs</u></b>), small insertions and deletions (<b><u>indels</u></b>) and copy number of variants (<b><u>CNVs</u></b>), reported in patients with psychiatric disorders. (Detail in Table 1)</p>
                <p><b>PsyMuKB</b> currently includes published <i>de novo</i> variants, both coding and non-coding, from <b>14</b> neuropsychiatric diseases, including Autism Spectrum Disorder (<b><u>ASD</u></b>), Bipolar Disorder (<b><u>BD</u></b>), Developmental Delay (<b><u>DD</u></b>), Intellectual Disability (<b><u>ID</u></b>), Obsessive Compulsive Disorder (<b><u>OCD</u></b>), Schizophrenia (<b><u>SCZ</u></b>), Tourette Syndrome (<b><u>TS</u></b>), Alzheimer early-onset (<b><u>AD</u></b>), etc., and unaffected siblings as control (<b><u>CTR</u></b>). (Detail in Table 1)</p>
                <p>In addition, large-scale genomics data is integrated into the PsyMuKB database including mRNA brain expression, transcript and regulatory element annotation, and protein-protein interactions (PPI). Thus, PsyMuKB provides users with the comprehensive exploration of the variants with the visualization, in terms of the brain expression pattern of the disrupted gene and/or isoform, its genomic location in transcripts, and PPIs of the impacted protein.</p>
                <img src="/static/image001.png" alt="" align="middle" />
                <p style="text-align: center"><small>Fig 1. Flowchart of the PsyMuKB database</small></p>

                <hr>
                <div class="row" >
                    <div class="accordion-heading" style="font-size: x-large;" id="collapse2">
                        2.Data Summary
                    </div>
                    <br/>
                    <div>
                    <p>At present, the PsyMuKB database contains various types of mutations and more than ~510,000 de novo variants (Table 1).</p>
                    <p>Table 1. Statistics of de novo mutations in PsyMuKB</p>
                <img src="/static/image003.png" alt="" />
                </div>
                    <br/>
                    <div>
                    <p>Table 2. The types of de novo variants</p>
                    <img src="/static/image005.png" alt="" />
                </div>
                </div>

                <div class="row">
                    <div class="accordion-heading" style="font-size: x-large;" id="collapse3">
                          3.Browse
                    </div>
                    <br/>
                    <p style="text-indent:50px;" id="collapse3.1">3.1 Browse by gene symbol</p>
                    <p style="text-indent:90px;">You can select the interested gene by browsing through the list of gene symbols.(Fig 3.)</p>
                    <br/>
                    <img src="/static/image007.png" alt="" />
                    <p>Fig 2. The interface of browsing by gene symbol</p>
                    <br/>
                    <p style="text-indent:50px;" id="collapse3.2">3.2 Browser by gene sets</p>
                    <p style="text-indent:90px;">You can select the interested gene by selecting symbols from different gene sets. PsyMuKB provides users with 16 different gene sets curated from literature that are associated with neuronal functional, developments or behaviors, etc.(Fig 4.)</p>
                    <br/>
                    <img src="/static/image009.png" alt="" />
                    <p>Fig 3. The interface of browsing by gene sets</p>
                </div>

                <div class="row">
                    <div id="collapse4">
                    <div class="accordion-heading" style="font-size: x-large;">
                         4.Search

                    </div></div>
                    <br/>
                    <div>
                        <p>There are numbers of options available for querying the PsyMuKB. </p>
                        <p style="text-indent:50px;">4.1 Quick Search:</p>
                        <p style="text-indent:90px;">You can search by gene symbol, gene ID, or chromosome coordinates  on “Home” page.</p><br/>
                    </div>
                    <br/>
                    <img src="/static/image010.png" alt="" />
                    <p>Fig 4. Quick Search interface</p>
                    <br/>
                    <p>2)Advance search:</p>
                    <p><u><i>Search by gene symbol or gene entrez ID (same as in Quick Search):</i></u></p>
                    <br/>
                    <img src="/static/image011.png" alt="" />
                    <p>Fig 5. Gene Search interface</p>
                    <br/>
                    <p><u><i>Search by DNMs:</i></u></p>
                    <p>One can search for more precise <i>de novo</i> mutations by indicating mutations type and disorder, with optional specification of chromosome number and coordinates.</p>
                    <br/>
                    <img  src="/static/image013.png" alt="" />
                    <p>Fig 6. DNM Search interface</p>
                    <br/>
                    <p><u><i>Search by CNVs:</i></u></p>
                    <p>One can search for more precise <i>de novo</i> copy number of variants by indicating mutations type and disorder, with optional specification of chromosome number and coordinates.</p>
                    <br/>
                    <img  src="/static/image015.png" alt="" />
                    <p>Fig 7. CNV Search interface</p>
                </div>

                <div class="row">
                    <div id="collapse5">
                    <div class="accordion-heading" style="font-size: x-large;">
                         5.Information related to the query results

                    </div>
                    </div>
                    <br/>
                    <p>The query results contain two major sections: "<b>Gene information</b>" and "<b>Mutation information</b>.</p>
                </div>

                <div class="row">
                <div class="accordion-heading" style="font-size: x-large;">
                         5.1 Gene information

                </div><br/>
                <p >There are five modules: basic information (Uniprot summary and Refseq summary), gene expression in specific tissue or cell types,“Mutation statics”,<b>"Transcript information"</b>, and<b>"Protein-protein interaction"</b>.</p>
                <p style="text-indent:50px;">5.1.1 Gene expression in specific tissue or cell types</p>
                <p style="text-indent:50px;">The brain expressions including Spatio-temporal human developmental brain expression (BrainSpan), Genotype-Tissue expression (GTEx), human brain single cell expression (Zhang et al., PMID: 25186741), adult mouse brain single cell expression (Saunders et al., PMID: 30096299).</p>
                <br/>
                <p>a</p><img src="/static/image017.png" alt="" align="absbottom "/>
                <p>b</p><img src="/static/image018.png" alt="" />
                <p>c</p><img src="/static/image019.png" alt="" />
                <p>d</p><img src="/static/image020.png" alt="" />
                <p>Fig 8.  Gene expression in specific tissue or cell types. (a) Tissue expression (Data source: GTEx V7). (b) Spatio-temporal Human Developmental Brain Expression (BrainSpan). (c) Human embryonic prefrontal cortex single cell expression. (d) Adult mouse Brain single cell expression.</p>
                <br/>
                <p style="text-indent:50px;">5.1.2 Neuropsychiatric disease related mutation information</p>
                <p style="text-indent:50px;">The “Mutations” part in gene information page provides the information of neuropsychaiatric diesases related to the certain gene mutations.</p>
                <br/>
                <img src="/static/image021.png" alt="" />
                <p>Fig 9. Gene mutation-Neuropsychiatric diease information table</p>
                <br/>
                <p style="text-indent:50px;">5.1.3 Transcript information</p>
                <p style="text-indent:50px;">The mutations are visualized on top of all the isoforms of impacted gene. Mutations are drawn on top of all the isoforms according to their chromosome locations. The mutation size has been adjusted for the visualization purpose. </p>
                <br/>
                <p style="text-indent:50px;">The information including <b>3 diagrams</b>: Visualization of regulatory element DNMs, Visualizations of de novo variants in transcripts, Visualizations of de novo CNVs. With similar modes of strcture, Visualization of regulatory element DNMs (Fig 11.) as example.</p>
                <br/>
                <img src="/static/image022.png" alt="" />
                <p>Fig 10. Visualization of regulatory element DNMs</p>
                <br/>
                <p>The tables of <b>Gencode</b> transcripts and <b>Refseq</b> transcripts are also shown in this part.</p>
                <br/>
                <img src="/static/image023.png" alt="" />
                <p>Fig 11. Gencode transcripts and Refseq transcripts table</p>
                <br/>
                <p style="text-indent:50px;">5.1.4 Protein-protein interaction</p>
                <p style="text-indent:50px;">Protein-protein interaction (PPI) data is extracted from Biogrid v3.5.165. The red node indicates the queried gene, and all other blue nodes are its interacting partners (Fig 13.).</p>
                <br/>
                <img src="/static/image024.png" alt="" />
                <p>Fig 12. PPI Network (Data source: BioGrid)</p>
                <br/>
                <p>Below PPI diagram, there is a table show all the genes which have PPI in the diagram.(Fig 14.)</p>
                <br/>
                <img src="/static/image025.png" alt="" />
                <p>Fig 13. PPI statistical table</p>
            </div>

                <div class="row">
                <div class="accordion-heading" style="font-size: x-large;">
                         5.2 Mutation information

                </div>
                <br/>
                <p style="text-indent:50px;">5.2.1 SNV mutation</p>
                <p style="text-indent:50px;">We collected and curated both coding and non-coding de novo mutations (in total of ~280,000) from ~70 all major WES, WGS and targeted sequencing studies on 14 neuropsychiatric disorders, including autism spectrum disorders (ASD), intellectual disability (ID), epileptic encephalopathy (EE), schizophrenia (SCZ), bipolar disorder (BPD), attention-deficit/hyperactivity disorder (ADHD), obsessive compulsive disorder (OCD), Congenital Heart Disease (CHD), Tourette Disorder (TD), and healthy siblings as controls (<b>Table 1</b>). Further, we included the mutation annotation scores from tools, such as SIFT, PolyPhen, CADD, DeepSea to annotate the pathogenicity of variants. In addition, PsyMuKB also provides the visualization of exact locations of these mutations on the isoforms of the impacted genes in “Transcript information” section.</p>
                <br/>
                <p>The mutation table shown as below: </p>
                <br/>
                <p>a</p><img src="/static/image026.png" alt="" />
                <p>b</p><img src="/static/image027.png" alt="" />
                <p>c</p><img src="/static/image028.png" alt="" />
                <p>Fig 14. The mutation table. (a) Coding de novo mutations. (b) Non-coding de novo variations.</p>
                <br/>
                <p style="text-indent:50px;">5.2.2 CNV mutation</p>
                <p style="text-indent:50px;">We collected and curated Copy Number Variants from 19 studies on four psychiatric disorders, including ASD, BD, ID/DD, SCZ and Control (<b>Table 1</b>).</p>
                <p style="text-indent:50px;">The mutation table shown as below:</p>
                <br/>
                <img src="/static/image029.png" alt="" />
                <p style="text-indent:50px;">Fig 15. De novo copy number variations<p>
                <br/>
                <p style="text-indent:90px;">5.2.3.1 mRNA-level impact</p>
                <p style="text-indent:90px;">Isoform expressions of the human brain were downloaded from GTEx portal (2016-01-15_v7_RNASeQCv1.1.8). Interface design provides the graphical view on the brain expressions of all the impacted transcripts owing this mutation, which can help user realize the consequence of isoform carrying a certain mutation in specific brain region.</p>
                <br/>
                <img src="/static/image030.png" alt="" />
                <p>Fig 16. mRNA-level impact</p>
                <br/>
                <p style="text-indent:90px;">5.2.3.2 Protein-level impact</p>
                <p style="text-indent:90px;">Protein isoform expressions in tissues were downloaded from Proteomics DB. Like mRNA-level, this interface design allowed users to explore gene products special expression patterns by selecting impacted and/or brain expressed protein.</p>
                <br/>
                <img src="/static/image031.png" alt="" />
                <p>Fig 17. Protein-level impact</p>

            </div>

                <div class="row">
                <div id="collapse6">
                <div class="accordion-heading" style="font-size: x-large;">
                         6.Download

                </div>
                </div>
                <br/>
                <p>PsyMuKB provides users with download option next to each data table and visualization.</p>
                <br/>

                <img  src="/static/image032.png" alt="" />

                <img src="/static/image033.png" alt="" />
                <p>Fig 18. Download button samples</p>

            </div>
            </div>

        </div>


</div>
{% endblock %}